src/codegen/ast.lisp

```diff
@@ -428,97 +430,111 @@ both CL namespaces appearing in NODE"
      (node-variables-g (node-bind-expr node) :variable-namespace-only variable-namespace-only)
      (node-variables-g (node-bind-body node) :variable-namespace-only variable-namespace-only))))
 
-(defmethod tc:apply-substitution (subs (node node-literal))
```

Comment:
I moved this method definition to the `transformations.lisp` file, for all `node`s at once using `traverse`

Reply:
Can you add a short comment in the code?

    ;;; tc:apply-substitutions used to be here, but ...